src/content/documentation/develop/browser-extension-development-tools.md

```diff
@@ -15,7 +15,7 @@ tags:
     translation,
     webextensions,
   ]
-contributors: [rebloor, hellosct1, ani-sha, ankushduacodes]
+contributors: [rebloor, mdnwebdocs-bot, hellosct1, ani-sha, ankushduacodes]
```

Comment:
I am confused: are we re-adding them?

Reply:
No, we shouldn't be doing that. Thought I'd fix this before committing.

Reply:
I am confused.